build-android-lib.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
```

Comment:
LGTM in overall.

However, please move this script into somewhere android-specific.

Reply:
moved to ```/api/android```